pdfium/responses/responses.go

```diff
@@ -33,17 +33,27 @@ type CharPosition struct {
 	Bottom float64
 }
 
+type FontInformation struct {
+	Size         float64 // Font size in points (also known as em)
+	SizeInPixels int     // Font size in pixels, only available when PixelPositions is used.
```

Comment:
maybe make this a pointer as it is not calculated and returning a 0 when we didn't calculate seems off.
eventhough 0 is an unrealistic value anyway

Reply:
Yeah sounds better indeed, changed it :)